### PR TITLE
chore(docs) : fixed several typos

### LINKED ISSUE #22241: Fix typos across project
## Description
A lot of typos across docs. 

### Expected result
Typos are resolved.

### Actual result
Typos are resolved

### PR DESCRIPTION
**Pull request by shivaylamba**
## Description
Fixed typos.

### CODE REVIEW

**Comment by shivaylamba** [comment]
Please review @pieh @jb-san

File: packages/gatsby-cli/src/reporter/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ const prematureEnd = () => {
     status: ActivityStatuses.Failed,
   })
 
-  interuptActivities()
+  interruptActivities()
```
Comment (pieh) [inline packages/gatsby-cli/src/reporter/index.js]: You need same change few lines below (Line 88)

**Comment by shivaylamba** [comment]
I have updated it @pieh

File: packages/gatsby-cli/src/reporter/index.js (hunk 1 above)
Comment (pvdz) [reply packages/gatsby-cli/src/reporter/index.js]: I thought I had checked them all :(

I hate that github doesn't allow an "expand all" so I can quickly check with the browser.

Thanks for checking! :+1:

**Pull request merged**